test/e2e/page-objects/pages/settings/privacy-settings.ts

```diff
@@ -183,9 +183,26 @@ class PrivacySettings {
     return (await this.driver.findElement(this.displayedSrpText)).getText();
   }
 
-  async openRevealSrpQuiz(): Promise<void> {
-    console.log('Open reveal SRP quiz on privacy settings page');
+  async openSRPList(): Promise<void> {
+    await this.driver.delay(1000);
```

Comment:
Nit: Feels weird to have a `delay` here. Should probably be closer to the caller.